Tasks/Gradle/gradletask.ts

```diff
@@ -64,7 +66,7 @@ if (isCodeCoverageOpted && inputTasks.indexOf('clean') == -1) {
     gb.arg('clean'); //if user opts for code coverage, we append clean functionality to make sure any uninstrumented class files are removed
 }
 
-gb.argString(tl.getInput('options', false));
```

Comment:
@ericsciple thanks for pointing. but looks like this the way node invoking in both Win, *nix
all of them added without quotes
`gradlew.bat clean build abc xyz`